docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -577,6 +577,11 @@ Removed Lookup Plugins
 
 * ``redis_kv`` use :ref:`redis <redis_lookup>` instead.
 
+Removed httpapi Plugins
```

Comment:
Did this actually happen in 2.9 (since it's already branched) or does this change need to go into `porting_guide_2.10.rst`?

Reply:
Yeah, this needs to be 2.10. Will update.